dev/io.openliberty.jakarta.rest.4.0.internal_fat_tck/bnd.bnd

```diff
@@ -0,0 +1,28 @@
+#*******************************************************************************
+# Copyright (c) 2024 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
```

Comment:
```suggestion
```

Not needed in new files

Reply:
I didn't explicitly says this, but of course this comment also applies to the other new files.